src/index.html

```diff
@@ -12,7 +12,49 @@
       href="./style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
-  </body>
+  <body></body>
+  <div class="container">
```

Comment:
The div element with the class 'container' should be inside the body tag. The body tag should contain all the contents of an HTML document, such as text, hyperlinks, images, tables, lists, etc.